composer.json

```diff
@@ -12,14 +12,20 @@
         }
     ],
     "require": {
-        "php": "^5.4|^7.0"
+        "php": "^7.0",
+        "psr/http-client": "^1.0",
+        "psr/http-factory": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
         "mockery/mockery": "~0.8",
-        "guzzlehttp/guzzle": "~5.0"
+        "guzzlehttp/guzzle": "~5.0",
+        "symfony/http-client": "^4.3",
```

Comment:
since this prevents usage on PHP5, this should be added "live" in the .travis/etc config

Reply:
I don't know where to do this, the doc make me feel like this is the solution, can you confirm please?

```yaml
language: php

php:
  - 5.4
  - 5.5
  - 5.6
  - 7.0
  - 7.1
  - 7.2
  - 7.3

sudo: false

cache:
  directories:
    - $HOME/.composer/cache

matrix:
  include:
    - php: hhvm
      dist: trusty
    - php: ['7.1', '7.2', '7.3']
      before_install: travis_retry composer require symfony/http-client
```

Reply:
You should be able to setup the CI on your repo to confirm